src/client.ts

```diff
@@ -419,6 +423,10 @@ export class Client {
         let headers = context.init.headers as Record<string, string>;
         headers['X-Adzerk-Sdk-Version'] = versionString;
 
+        if (adzerkApiKey) {
+          headers['X-Adzerk-ApiKey'] = adzerkApiKey;
```

Comment:
This can actually be `X-Kevel-ApiKey`. We support both. Suggest moving to the new value. Probably should rename the variables as well.

Reply:
Waiting until we rename everything to kevel to keep consistency